pss/trojan/message.go

```diff
@@ -45,6 +45,12 @@ const MaxPayloadSize = 4030
 
 var hashFunc = storage.MakeHashFunc(storage.BMTHash)
 
+// RecoveryTopicText is the string used to construct the RecoveryTopic var
+const RecoveryTopicText = "RECOVERY"
```

Comment:
we need a separate string here as a `const` in order to build a `feed.Topic` var in the `getPinners` function (which is _not_ of the `trojan.Topic` type)

Reply:
this should not be here. trojan pkg not supposed to know about recovery

Reply:
thanks, moved this to `prod`